src/core/scope.ml

```diff
@@ -426,24 +339,24 @@ let scope : mode -> sig_state -> env -> p_term -> tbox = fun md ss env t ->
     | (P_LLet(_)       , M_Patt             ) ->
         fatal t.pos "Let-bindings are not allowed in a pattern."
     | (P_NLit(n)       , _                  ) ->
-        let sym_z = _Symb (Builtin.get t.pos ss.builtins "0") Nothing
-        and sym_s = _Symb (Builtin.get t.pos ss.builtins "+1") Nothing in
+        let sym_z = _Symb (Builtin.get ss t.pos "0")
+        and sym_s = _Symb (Builtin.get ss t.pos "+1") in
         let rec unsugar_nat_lit acc n =
           if n <= 0 then acc else unsugar_nat_lit (_Appl sym_s acc) (n-1)
         in
         unsugar_nat_lit sym_z n
     | (P_UnaO(u,t)    , _                   ) ->
         let (s, impl) =
-          let (op,_,qid) = u in
-          let (s, _) = find_sym ~prt:true ~prv:true true ss qid in
-          (_Symb s (Unary(op)), s.sym_impl)
+          let (_op,_,qid) = u in
```

Comment:
Why??? I find this feature very useful.

Reply:
Because that make harder to parse when you look over the code quickly. We don't need it, there is no reason to name it. The underscore disables the warning but does not prevent someone from using the value in the code. This is just bad style.